.gitignore

```diff
@@ -2,3 +2,4 @@
 tmp
 dist/
 build
+.idea
```

Comment:
Can you set your editor (which I guess is goland?) to append a newline at the end of the file when you save. Also preferable sort the whole `.gitignore`.